livekit-agents/livekit/agents/tokenize/basic.py

```diff
@@ -34,7 +34,7 @@ def __init__(
             stream_context_len=stream_context_len,
         )
 
-    def tokenize(self, *, text: str, language: str | None = None) -> list[str]:
+    def tokenize(self, text: str, language: str | None = None) -> list[str]:
```

Comment:
this is a bit inconsistent with `stream`, where language is required to be a kwarg.. I would say let's be consistent here.

Reply:
ah, language should be removed from there actually, the right place is the constructor

Reply:
will follow the STT API